extension/src/popup/views/Debug.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+// Remove when not needed! 👇
+// eslint-disable-next-line arrow-body-style
```

Comment:
I figure this component would get something added that would necessitate `{}` basically the next time it's touched, so why force whitespace changes